src/main/java/net/glowstone/command/minecraft/BanListCommand.java

```diff
@@ -0,0 +1,64 @@
+package net.glowstone.command.minecraft;
+
+import net.glowstone.command.CommandUtils;
+import org.bukkit.BanEntry;
+import org.bukkit.BanList;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.defaults.VanillaCommand;
+import org.bukkit.util.StringUtil;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class BanListCommand extends VanillaCommand {
+
+    private static final List<String> BAN_TYPES = Arrays.asList("ips", "players");
+
+    public BanListCommand() {
+        super("banlist", "Displays the server's blacklist.", "/banlist [ips|players]", Collections.emptyList());
+        setPermission("minecraft.command.ban.list");
+    }
+
+    @Override
+    public boolean execute(CommandSender sender, String commandLabel, String[] args) {
+        if (!testPermission(sender)) return false;
+
+        BanList.Type banType;
+
+        if (args.length > 0) {
+            if ("ips".equalsIgnoreCase(args[0])) {
+                banType = BanList.Type.IP;
+            } else if ("players".equalsIgnoreCase(args[0])) {
+                banType = BanList.Type.NAME;
+            } else {
+                sender.sendMessage(ChatColor.RED + "Invalid parameter '" + args[0] + "'. Usage: " + usageMessage);
+                return false;
+            }
+        } else {
+             banType = BanList.Type.NAME;
+        }
+
+        final Set<BanEntry> banEntries = Bukkit.getBanList(banType).getBanEntries();
+
+        if (banEntries.isEmpty()) {
+            sender.sendMessage("There are no banned players");
+        } else {
+            final List<String> targets = banEntries.stream().map(BanEntry::getTarget).collect(Collectors.toList());
+            sender.sendMessage("There are " + banEntries.size() + " banned players: ");
+            sender.sendMessage(CommandUtils.prettyPrint(targets.toArray(new String[targets.size()])));
```

Comment:
@Mystiflow That check is done here: https://github.com/GlowstoneMC/Glowstone/pull/518/files/946cda8cadfbdf72f3a29d6a4fae84f80276bb56..779d13ed59cef62cd218cdc6d6fed30cb814e40f#diff-d9df5a37e1afb33765f601eec8d00d0aL55